src/averageAges.js

```diff
@@ -55,7 +67,25 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const womenArr
+    = people.filter(item => item.sex === 'f'
+    && people.some(child => item.name === child.mother));
+
+  const childrenArr = onlyWithSon
```

Comment:
```suggestion
  const children = onlyWithSon
```
If a variable's name is in plural form, it's obvious that it's an array. No need to include 'arr' in its name